webpack.config.production.js

```diff
@@ -1,20 +1,23 @@
 'use strict';
 
+var _ = require('lodash');
 var webpack = require('webpack');
 var baseConfig = require('./webpack.config.base');
 
-var config = Object.create(baseConfig);
-config.plugins = [
-  new webpack.optimize.OccurenceOrderPlugin(),
-  new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify('production')
-  }),
-  new webpack.optimize.UglifyJsPlugin({
-    compressor: {
-      screw_ie8: true,
-      warnings: false
-    }
-  })
-];
-
-module.exports = config;
+module.exports = _.merge({}, baseConfig, {
+  plugins: _.union(baseConfig.plugins, [
+    new webpack.DefinePlugin({
+      '__DEV__': 'false',
+      'process.env.NODE_ENV': '"production"'
+    }),
+    new webpack.optimize.UglifyJsPlugin({
+      compressor: {
```

Comment:
Oh good. I was always curious what you're referred to in some older tweet about not going all the way with Uglify. :-)